README.md

```diff
@@ -86,14 +86,15 @@ environment:
 - [VirtualBox Download](https://www.virtualbox.org/wiki/Downloads)
 - [Ansible Download](http://docs.ansible.com/ansible/latest/intro_installation.html) 
 
-Once you have these tools installed, you will need to download the Ansible
-roles used to build the Vagrant instance. Run the following in the ``deploy``
-folder in directory where you have cloned Ocim:
+Once you have these tools installed, you will need to download the [Ansible
+playbooks](https://github.com/openc-craft/ansible-playbooks) used to build the
+Vagrant instance.  If you haven't checked it out yet, you can clone it into the
+`deploy/` subdirectory using this command at the root of the repository:
 
-    ansible-galaxy install -r requirements.yml
+    git clone https://github.com/openc-craft/ansible-playbooks/deploy
```

Comment:
`openc-craft` -> `open-craft`

Reply:
I think you meant to replace the `/` between `ansible-playbooks` and `deploy` with a space.

Reply:
Also, because we're running the playbook from the host, we actually need the correct Ansible 2.5 install through `deploy/requirements.txt`, so that should be another step.

Reply:
The installation of Ansible is mentioned above, as it hass been before my change.  This actually has always been somewhat out of order, so I fixed it now.